### PR TITLE
Some multilanguage support improvements

### PR DESCRIPTION
**Pull request by lomion0815**
I changed some things in order to improve multilanguage support. The language can be switch with a link in the footer.

### CODE REVIEW

**Comment by matsuyoshi30** [comment]
@lomion0815 Thanks for your PR and comments @mkalus 's PR #78. mkalus updated (, created another) PR and I've just merged it, so please resolve conflict in `i18n/de.toml`. I'll take a look soon!

**Comment by lomion0815** [comment]
> @lomion0815 Thanks for your PR and comments @mkalus 's PR #78. mkalus updated (, created another) PR and I've just merged it, so please resolve conflict in `i18n/de.toml`. I'll take a look soon!

Should be fine now.

**Comment by mkalus** [comment]
Sorry, missed this pull request.

File: layouts/_default/single.html [hunk 1]
```diff
@@ -6,8 +6,9 @@
       {{ if .Params.tags }}
         <div class="blog-tags">
           {{ range .Params.tags }}
-            <a href="{{ $.Site.BaseURL }}/tags/{{ . | urlize }}/">{{ . }}</a
+            <a href="{{ "" | absLangURL }}/tags/{{ . | urlize }}/">{{ . }}</a
             >&nbsp;
+            <a href="{{ "" | absLangURL }}/tags/{{ . | urlize }}/">{{ . }}</a>&nbsp;
```
Comment (matsuyoshi30) [inline layouts/_default/single.html:11]: I think this line is the same as above the line (L9), so it doesn't need.

**Comment by matsuyoshi30** [comment]
@lomion0815 Thanks for your PR and sorry for late my response. I commented on one thing, so please check it!

File: layouts/_default/single.html (hunk 1 above)
Comment (lomion0815) [reply layouts/_default/single.html:11]: You are right. Only different line break. Just remove it ...

**Comment by matsuyoshi30** [comment]
@lomion0815 LGTM. Thank you again!

**Pull request merged**